app/editinplace.php

```diff
@@ -111,19 +111,12 @@
                 // Give the message a subject
                 ->setSubject(_('[eLabFTW] New comment posted'))
                 // Set the From address with an associative array
-                ->setFrom(array(get_config('smtp_username') => get_config('smtp_username')))
+                ->setFrom(get_config('mail_from'))
```

Comment:
For consistency reasons, yes. Would work without array as well. Changed in https://github.com/Athemis/elabftw/commit/ea2eeddd47f696fca69404476924d6d9dd0252a0